srcpkgs/rtkit/template

```diff
@@ -1,10 +1,11 @@
 # Template file for 'rtkit'
 pkgname=rtkit
 version=0.13
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="pkg-config"
 makedepends="dbus-devel libcap-devel"
+depends="dbus polkit>=0.92"
```

Comment:
I don't think this needs to be versioned... We haven't had that version for a long time now.

Reply:
I put it just because it was on the readme